src/github.com/getlantern/flashlight/util/common.go

```diff
@@ -0,0 +1,58 @@
+package util
+
+import (
+	"errors"
+	"github.com/getlantern/golog"
+	"os"
+	"os/user"
+	"path"
+)
+
+var (
+	log = golog.LoggerFor("util")
+)
+
+type logger func(arg interface{})
+
+func GetUserHomeDir() (string, error) {
+	usr, err := user.Current()
+	if err != nil {
+		log.Errorf("Error locating user home directory %s", err)
+		return "", err
+	}
+	lanternDir := path.Join(usr.HomeDir, ".lantern")
+	// Create the ~/.lantern directory if it doesn't exist already
+	exists, _ := DirExists(lanternDir)
+	if !exists {
+		err = os.Mkdir(lanternDir, 0755)
+		if err != nil {
+			log.Errorf("Error creating user home directory: %s", err)
```

Comment:
This could be misleading, as we're creating the lantern dir and not the user dir.